realsense_camera/src/base_nodelet.cpp

```diff
@@ -275,34 +276,67 @@ namespace realsense_camera
   {
     // print list of detected cameras
     std::vector<int> camera_type_index;
-    std::string detected_camera_msg = " - Detected the following cameras:";
+
+    // map camera names to validated firmware
+    std::map<std::string, std::string> camera_name_to_validated_firmware =
```

Comment:
When I put the map declaration/initialization into constants.h, I get the following error:

```
CMakeFiles/realsense_camera_nodelet.dir/src/r200_nodelet.cpp.o:(.bss+0x0): multiple definition of `realsense_camera::camera_name_to_validated_firmware[abi:cxx11]'
CMakeFiles/realsense_camera_nodelet.dir/src/base_nodelet.cpp.o:(.bss+0x0): first defined here
CMakeFiles/realsense_camera_nodelet.dir/src/f200_nodelet.cpp.o:(.bss+0x0): multiple definition of `realsense_camera::camera_name_to_validated_firmware[abi:cxx11]'
CMakeFiles/realsense_camera_nodelet.dir/src/base_nodelet.cpp.o:(.bss+0x0): first defined here
CMakeFiles/realsense_camera_nodelet.dir/src/sr300_nodelet.cpp.o:(.bss+0x0): multiple definition of `realsense_camera::camera_name_to_validated_firmware[abi:cxx11]'
CMakeFiles/realsense_camera_nodelet.dir/src/base_nodelet.cpp.o:(.bss+0x0): first defined here
CMakeFiles/realsense_camera_nodelet.dir/src/zr300_nodelet.cpp.o:(.bss+0x0): multiple definition of `realsense_camera::camera_name_to_validated_firmware[abi:cxx11]'
CMakeFiles/realsense_camera_nodelet.dir/src/base_nodelet.cpp.o:(.bss+0x0): first defined here
collect2: error: ld returned 1 exit status
realsense/realsense_camera/CMakeFiles/realsense_camera_nodelet.dir/build.make:465: recipe for target '/home/arbrindl/catkin_ws/devel/lib/librealsense_camera_nodelet.so' failed
make[2]: *** [/home/arbrindl/catkin_ws/devel/lib/librealsense_camera_nodelet.so] Error 1
CMakeFiles/Makefile2:2383: recipe for target 'realsense/realsense_camera/CMakeFiles/realsense_camera_nodelet.dir/all' failed
make[1]: *** [realsense/realsense_camera/CMakeFiles/realsense_camera_nodelet.dir/all] Error 2
Makefile:138: recipe for target 'all' failed
make: *** [all] Error 2
Invoking "make -j8 -l8" failed
```

I'm not sure why it complains about multiple definitions of the map when I don't use it in anything other than base_nodelet.cpp. I was only able to get rid of these errors when I put the map in base_nodelet.cpp instead of constants.h. Do you have any idea why these errors would pop up when I move the map to constants.h?